widgets/src/main/java/org/pentaho/gwt/widgets/client/utils/string/CssUtils.java

```diff
@@ -0,0 +1,358 @@
+/*
```

Comment:
I don't know what is the proper way of adding this kind of code in our repositories, but shouldn't this be a Pentaho license header, taking into account that we adapted it to our use case?

Reply:
In this same project, there are several examples, taking different approaches.
* A - Mixed header - https://github.com/pentaho/pentaho-commons-gwt-modules/blob/master/widgets/src/main/java/org/pentaho/gwt/widgets/client/utils/CronExpression.java#L2
* B - Original header - https://github.com/pentaho/pentaho-commons-gwt-modules/blob/master/widgets/src/main/java/org/pentaho/gwt/widgets/client/datepicker/PentahoMonthSelector.java#L2

It may depend on the amount of customization. In our case, there was no "customization", but simply removal of any code not applicable to our use case. Thus, I chose the latter.

Reply:
I'm thinking of asking for IP to review this explicitly, in the final artifact, by asking for it in the `#oss-reports` channel.